specs-go/config.go

```diff
@@ -430,6 +430,8 @@ type SolarisAnet struct {
 
 // Windows defines the runtime configuration for Windows based containers, including Hyper-V containers.
 type Windows struct {
+	// LayerFolders contains a list of absolute paths to directories containing image layers.
+	LayerFolders []string `json:"layerFolders"`
```

Comment:
@wking @jhowardmsft i modified the field name as per [this suggestion](https://github.com/opencontainers/runtime-spec/pull/828#discussion_r117719673), should I revert it back?

It looks like [there are linux fields that use camel case as well](https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#user-namespace-mappings).